packages/react/src/Column/Column.test.tsx

```diff
@@ -19,7 +19,7 @@ describe('Column', () => {
     const component = container.querySelector(':only-child')
 
     expect(component).toHaveClass('ams-column')
-    expect(component).toHaveClass('ams-column--gap-medium')
+    expect(component).not.toHaveClass('ams-column--gap-medium')
```

Comment:
So to continue my rant. This should be true if medium is the default.

Reply:
No, this class name no longer exists.

The test is not really necessary to be honest – before today it would have been `ams-column--medium`. Should I update or remove the test?

Reply:
It is a little weird to test for something that will never exist. Let's remove it

Reply:
They’re gone.